src/tslint.json

```diff
@@ -1,3 +1,18 @@
 {
-  "extends": "../tslint.json"
+  "extends": "../tslint.json",
+  "rulesDirectory": [
+    "../node_modules/codelyzer"
+  ],
+  "rules": {
+    "template-accessibility-alt-text": true,
+    "template-accessibility-elements-content": true,
+    "template-accessibility-label-for": true,
+    "template-accessibility-tabindex-no-positive": true,
+    "template-accessibility-table-scope": true,
+    "template-accessibility-valid-aria": true,
+    "template-click-events-have-key-events": true,
```

Comment:
I was more thinking about deactivating / removing `template-click-events-have-key-events` and `template-mouse-events-have-key-events` completely from our configuration, not disabling them on per-widget basis.

Reasoning being that rule is too simplistic for us, we don't add event handlers using the `(keyup)`, but manually with typescript. Also I don't really understand why enforce this rule on links and buttons (aslo often the case in ng-bootstrap)...

WDYT?

Reply:
> 
> 
> I was more thinking about deactivating / removing `template-click-events-have-key-events` and `template-mouse-events-have-key-events` completely from our configuration, not disabling them on per-widget basis.
> 
> Reasoning being that rule is too simplistic for us, we don't add event handlers using the `(keyup)`, but manually with typescript. Also I don't really understand why enforce this rule on links and buttons (aslo often the case in ng-bootstrap)...
> 
> WDYT?

The rule is too simplistic, that's true. However, it's still a reminder that for every click handler there should be a keyboard handler. Personally, I prefer disabling the rule per case than accidentally creating an inaccessible component. WDYT?

Reply:
Well we don't create components without keyboard support here anyway :) To me, all 4 cases it reported in ng-bootstrap were completely invalid and the less case-by-case exceptions we have - the better.

Reply:
After going through all the rules in codelyzer, I'm not at all convinced they're very useful in our library scope :/ It's more for the end apps from what I feel...